appengine/flexible/SendGrid/README.md

```diff
@@ -0,0 +1,53 @@
+# Send Grid and Google App Engine Flexible Environment
+
+This sample application demonstrates how to send email with Send Grid
+when running in Google App Engine Flexible Environment.
+
+## Prerequisites
+
+1.  **Follow the instructions in the [root README](../../../README.md).**
+  
+2.  Install the [Google Cloud SDK](https://cloud.google.com/sdk/).
+    The Google Cloud SDK is required to deploy .NET applications to App Engine.
+
+3.  (Optional) Visual Studio is not required, but
+    [Google Cloud Tools for Visual Studio](
+        https://marketplace.visualstudio.com/items?itemName=GoogleCloudTools.GoogleCloudPlatformExtensionforVisualStudio)
+    make it easy to deploy to App Engine.  Install them if you are running 
+    Visual Studio.
+
+4.  [Create a SendGrid account](http://sendgrid.com/partner/google).
+
+4.  Edit [appsettings.json](appsettings.json).  Replace 
+    `your-sendgrid-api-key` with your sendgrid api key.
+
+## Run Locally
+
+### Using PowerShell
+```psm1
+PS > dotnet restore
+PS > dotnet run
+```
+
+### Using Visual Studio
+Press F5.
```

Comment:
1.  In Solution Explorer, right-click the **SendGrid** project and choose **Set as StartUp Project**

2. Press F5.